index.js

```diff
@@ -293,6 +293,7 @@ class MillerColumnsElement extends HTMLElement {
       columnCollapse: 'miller-columns__column--collapse',
       columnMedium: 'miller-columns__column--medium',
       columnNarrow: 'miller-columns__column--narrow',
+      columnActive: 'miller-columns__column--active',
```

Comment:
This code looks good, but could the commit be split into two?

>Identify the active column based on the active item and refactor column styling to only show the active column, full width, on mobile devices.

=>

1) `Identify the active column based on the active item`
2) `Refactor column styling to only show the active column, full width, on mobile devices`

Reply:
Split this into separate commits as suggested.

Reply:
Thanks.